functionary/core/migrations/0001_initial.py

```diff
@@ -0,0 +1,420 @@
+                        serialize=False,
+                    ),
+                ),
+                ("name", models.CharField(db_index=True, max_length=64, unique=True)),
+            ],
+            bases=(core.models.mixins.ModelSaveHookMixin, models.Model),
+        ),
+        migrations.CreateModel(
+            name="TaskLog",
+            fields=[
+                (
+                    "task",
+                    models.OneToOneField(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        primary_key=True,
+                        serialize=False,
+                        to="core.task",
+                    ),
+                ),
+                ("log", models.TextField()),
+                ("created_at", models.DateTimeField(auto_now_add=True)),
+            ],
+        ),
+        migrations.CreateModel(
+            name="TaskResult",
+            fields=[
+                (
+                    "task",
+                    models.OneToOneField(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        primary_key=True,
+                        serialize=False,
+                        to="core.task",
+                    ),
+                ),
+                ("result", models.TextField()),
+                ("created_at", models.DateTimeField(auto_now_add=True)),
+            ],
+        ),
+        migrations.CreateModel(
+            name="TeamUserRole",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                (
+                    "role",
+                    models.CharField(
+                        choices=[
+                            ("ADMIN", "admin"),
+                            ("DEVELOPER", "developer"),
+                            ("OPERATOR", "operator"),
+                            ("READ_ONLY", "read_only"),
+                        ],
+                        max_length=64,
+                    ),
+                ),
+                ("updated_at", models.DateTimeField(auto_now=True)),
+                (
+                    "team",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        related_name="user_roles",
+                        to="core.team",
+                    ),
+                ),
+                (
+                    "user",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        related_name="%(class)ss",
+                        to=settings.AUTH_USER_MODEL,
+                    ),
+                ),
+            ],
+        ),
+        migrations.CreateModel(
```

Comment:
It looks like this migration was generated before rebasing onto main with the changed from #73.  I had to run `makemigrations` and `migrate` again to get the new summary field added.

Reply:
Included missing migration in newest commit